assets/queries/ansible/hosts/ansible_tower_exposed_to_internet/query.rego

```diff
@@ -0,0 +1,21 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	doc := input.document[id].all
+	doc.children.tower.hosts[ip]
+
+    not common_lib.isPrivateIP(ip)
+
+	result := {
+		"documentId": input.document[id].id,
+		"hostname": "tower",
+		"resourceName": "children",
+		"resourceType": "n/a",
+		"searchKey": "all.children.tower.hosts",
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "Ansible Tower IP should be private",
+		"keyActualValue": "Ansible Tower IP is public",
```

Comment:
```suggestion
		"keyActualValue": "Ansible Tower IP is public",
		"searchLine": common_lib.build_search_line(["tower", "hosts"], [ip]),
```
Please add the search line and make sure that the vulnerability builder is capable of using it
I have tried `"searchLine": common_lib.build_search_line(["tower", "hosts"], [ip]),` and `"searchLine": common_lib.build_search_line(["all","children","tower", "hosts"], [ip]),` and none work.